packages/expo-dev-launcher/android/src/debug/java/expo/modules/devlauncher/modules/DevLauncherInternalModule.kt

```diff
@@ -100,7 +100,11 @@ class DevLauncherInternalModule(reactContext: ReactApplicationContext?)
 
   @ReactMethod
   fun getPendingDeepLink(promise: Promise) {
-    promise.resolve(intentRegistry.intent?.data?.toString())
+    intentRegistry.intent?.data?.let {
+      promise.resolve(it.toString())
+    }
+
+    promise.resolve(intentRegistry.intent?.action)
```

Comment:
I don't think we should call `resolve` twice on the same Promise, even if it happens to be a no-op the second time. It would be more future-proof to be explicit about what we do and don't want to happen in each case.

Reply:
Good find 💪